### PR TITLE
Update example commands in README

### PR DESCRIPTION
**Pull request by Alexsandruss**
## Description

- Change example commands to frequently used ones.

---

PR should start as a draft, then move to ready for review state after CI is passed and all applicable checkboxes are closed.
This approach ensures that reviewers don't spend extra time asking for regular requirements.

You can remove a checkbox as not applicable only if it doesn't relate to this PR in any way.

Checklist to comply with **before moving PR from draft**:

**PR completeness and readability**

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes or created a separate PR with update and provided its number in the description, if necessary.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_.
- [x] I have added a respective label(s) to PR if I have a permission for that.
- [x] I have resolved any merge conflicts that might occur with the base branch.

**Testing**

- [x] I have run it locally and tested the changes extensively.
- [x] All CI jobs are green or I have provided justification why they aren't.
- [x] I have extended testing suite if new functionality was introduced in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,22 +44,41 @@ conda env create -n rapids --solver=libmamba -f envs/conda-env-rapids.yml
...
 ```bash
-python -m sklbench --config configs/sklearn_example.json --report
+python -m sklbench -c configs/regular \
+    -f algorithm:library=sklearnex algorithm:device=cpu \
+    -e ENV_NAME -r result_sklearnex_cpu_regular.json \
+    --report --report-file result-sklearnex-cpu-regular.xlsx
 ```
 
-By default, output and report file paths are `result.json` and `report.xlsx`. To specify custom file paths, run:
+In order to optimize datasets downloading and get more verbose output, use `--prefetch-datasets` and `-l INFO` arguments:
```
Comment (david-cortes-intel) [inline README.md:68]: Please remember to provide info about the requirements for kaggle data.

File: README.md [hunk 2]
```diff
@@ -44,22 +44,41 @@ conda env create -n rapids --solver=libmamba -f envs/conda-env-rapids.yml
...
 
 ```bash
-python -m sklbench --config configs/sklearn_example.json
+python -m sklbench --configs configs/regular \
+    --filters algorithm:library=sklearnex algorithm:device=cpu \
+    --environment-name ENV_NAME --result-file result_sklearnex_cpu_regular.json
+# Same command with shorter argument aliases for typing convenience
+python -m sklbench -c configs/regular \
+    -f algorithm:library=sklearnex algorithm:device=cpu \
+    -e ENV_NAME -r result_sklearnex_cpu_regular.json
```
Comment (david-cortes-intel) [inline README.md:56]: This `ENV_NAME` is very unclear to me.

The docs say:
> Environment name to use instead of it's configuration hash.

But that doesn't tell me what the environment is or what is is used for. Should `ENV_NAME` be substituted with something else? Is it required?

File: README.md [hunk 3]
```diff
@@ -69,7 +88,9 @@ For a description of all benchmarks runner arguments, refer to [documentation](s
 To combine raw result files gathered from different environments, call the report generator:
 
 ```bash
-python -m sklbench.report --result-files result_1.json result_2.json --report-file report_example.xlsx
+python -m sklbench.report \
```
Comment (david-cortes-intel) [inline README.md:91]: I think this one is missing the flag that's needed when mixing sklearn and sklearnex.

File: README.md [hunk 4]
```diff
@@ -44,22 +44,41 @@ conda env create -n rapids --solver=libmamba -f envs/conda-env-rapids.yml
...
+    -e ENV_NAME -r result_sklearnex_cpu_regular.json \
+    --report --report-file report-sklearnex-cpu-regular.xlsx \
+    --prefetch-datasets -l INFO
+```
 
+To select measurement for few algorithms only, extend filter (`-f`) argument:
 ```bash
-python -m sklbench --config configs/sklearn_example.json --report --result-file result_example.json --report-file report_example.xlsx
+# ...
+  -f algorithm:library=sklearnex algorithm:device=cpu algorithm:estimator=PCA,KMeans
```
Comment (david-cortes-intel) [inline README.md:80]: Here it could mention that these algorithms need to be in the config JSON.

File: README.md [hunk 5]
```diff
@@ -44,22 +44,41 @@ conda env create -n rapids --solver=libmamba -f envs/conda-env-rapids.yml
...
+In order to optimize datasets downloading and get more verbose output, use `--prefetch-datasets` and `-l INFO` arguments:
+```bash
+python -m sklbench -c configs/regular \
+    -f algorithm:library=sklearnex algorithm:device=cpu \
+    -e ENV_NAME -r result_sklearnex_cpu_regular.json \
+    --report --report-file report-sklearnex-cpu-regular.xlsx \
+    --prefetch-datasets -l INFO
+```
 
+To select measurement for few algorithms only, extend filter (`-f`) argument:
```
Comment (david-cortes-intel) [inline README.md:77]:
```suggestion
To run benchmarks for a for few algorithms only, extend filter (`-f`) argument:
```

File: README.md (hunk 1 above)
Comment (Vika-F) [inline README.md:68]: Why not to use `--prefetch-datasets` as the default recommendation for regular runs?
Does it have any drawbacks?
